extensions/sonarqube/overview.md

```diff
@@ -1,50 +1,59 @@
+
+![Build Config](img/Build-pipeline.png)
+
+When a build is run on a branch, the extension automatically configures the analysis to be pushed to the relevant project branch on SonarQube.
 
 ![PR-Decoration](img/pull-request-decoration.png)
 
 **Important note**: to activate pull request decoration, you must specify a user token in the "General Settings > Pull Requests" administration page of your project in SonarQube.
 
-### Quality Gate Status
-The **Publish Quality Gate Result** task waits for the analysis report to be consumed by the SonarQube in order to flag the build job with the Quality Gate status. The Quality Gate is a major, out-of-the-box, feature of SonarQube. It provides the ability to know at each analysis whether an application passes or fails the release criteria. In other words it tells you at every analysis whether an application is ready for production "quality-wise".
+## Additional Highlights
+### Seamless integration with .NET solutions
+C# and VB.NET analysis is simple and straightforward and only requires adding the **Prepare Analysis Configuration** and **Run Code Analysis** tasks to your build definition.
 
-Example of a passing Quality Gate:
-![Passed Qualiy Gate](img/sq-analysis-report-passed.png)
+### Easy setup for Maven and Gradle projects
+For Java, analyzing your source code is also very easy. It only requires adding the **Prepare Analysis Configuration** task and checking the **Run SonarQube Analysis** option in the 'Code Analysis' panel in your Maven or Gradle task.
 
-Example of a failing Quality Gate:
-![Failed Qualiy Gate](img/sq-analysis-report-failed.png)
+## SonarQube Azure DevOps Marketplace Extension Details
+This extension provides the following features:
+* A dedicated **SonarQube EndPoint** that defines the SonarQube server to be used.
+* Branch and **Pull Request** analysis in your projects.
+* Three build tasks to get your projects analyzed easily:
+  * **Prepare Analysis Configuration:** Configures all the required settings prior to executing a build. This task is mandatory. For .NET solutions or Java projects, this task helps SonarQube seamlessly integrate with MSBuild, Maven and Gradle tasks.
+  * **Run Code Analysis:** Executes the source code analysis. This task isn't required for Maven or Gradle projects.
+  * **Publish Quality Gate Result:** Displays the Quality Gate status in the build summary. This tasks is optional, as it may increase the overall build time.
 
+**Note for TFS installations older than TFS 2017 Update 2**: to install the extension, please follow instructions
+available on the ["SonarQube Extension 3.0" documentation page](https://docs.sonarqube.org/display/SCAN/SonarQube+Extension+3.0).
```

Comment:
There's a line return between instructions and available.
Also, I can't find anymore the documentation page for v3. This link fallbacks on same page as the [getstarted] link provided below.  I guess that this was done on purpose and we can drop the note.